recipes/proj/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibaecConan(ConanFile):
+    name = "proj"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://proj.org"
+    description = "Cartographic Projections and Coordinate Transformations Library"
+    topics = ("conan", "dsp", "proj", "proj4", "projections", "gis", "geospatial")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt", "sqlite_init.py", "patches/*"]
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("sqlite3/3.31.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
```

Comment:
Recently, I've started to cache `autotools` and `cmake` objects inside the `ConanFile` object.
This avoid a reconfiguration when doing `build()` and `package()`.

Reply:
Easy enough.  Made the change and will keep it in mind for future package.